commit 1967aa4e361998676ccff4f29dcc70a10fd668e1 (refs/changes/61/1161/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-12-22T00:54:57+00:00 (2 years, 10 months ago)
    
    MANTA-3066 bump cueball dep for cueball#63 fix
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 97e78da..b9a9eae 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "backoff": "2.3.0",
         "bunyan": "0.22.1",
         "bunyan-syslog": "0.2.2",
-        "cueball": "1.2.1",
+        "cueball": "1.2.2",
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
         "dtrace-provider": "0.2.8",
